task_26/andrew.samsonov/src/main/java/ua/pp/darknsoft/jdbc/DBUtil.java

```diff
@@ -0,0 +1,96 @@
+package ua.pp.darknsoft.jdbc;
+
+import ua.pp.darknsoft.Main;
+import ua.pp.darknsoft.entity.Author;
+import ua.pp.darknsoft.entity.Book;
+import ua.pp.darknsoft.entity.Consumer;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class DBUtil {
+
+  private DBUtil() {
+  }
+
+  public static Connection getConnection() throws SQLException {
```

Comment:
It might be a good idea, to restrict calling *getConnection()* function many times, since in our program we need to create a new connection only once. One way to do it is to create a singleton out of DBUtil class.